plugins/chat/assets/javascripts/discourse/components/chat/form.gjs

```diff
@@ -2,13 +2,13 @@ import Component from "@glimmer/component";
 import ChatFormSection from "discourse/plugins/chat/discourse/components/chat/form/section";
 
 export default class ChatForm extends Component {
```

Comment:
Huh! Do you know why these class-member-ordering issues weren't picked up with the existing config? 🤔

Reply:
The rule seems pretty broken still, I had a few places where it specifically asked me to put the template tag after the properties and service injections but specifically before the getters. I think that’s probably why I put it at the top initially because that seemed most consistently reliable. Maybe that is actually working now?

Reply:
Nop, it still doesn't work - I modified those by hand 😅 I'll add a test to `@discourse/lint-configs` covering this bug but in the meantime we'll have to live with it 🥲